docs/index.md

```diff
@@ -0,0 +1,107 @@
+<p align="center">
+  <a href="https://reactionmechanismgenerator.github.io/T3/">
+  <img src="T3_logo_small.gif" alt="T3"></a>
+</p>
+<p align="center">
+    <em>The Tandem Tool for automated kinetic model generation and refinement</em>
+</p>
+<p align="center">
+<a href="https://github.com/ReactionMechanismGenerator/T3/releases" target="_blank">
+    <img src="https://img.shields.io/badge/version-0.1.0-blue.svg" alt="Release">
+</a>
+<a href="https://travis-ci.com/ReactionMechanismGenerator/T3" target="_blank">
+    <img src="https://travis-ci.com/ReactionMechanismGenerator/T3.svg?branch=master" alt="Build Status">
+</a>
+<a href="https://lgtm.com/projects/g/ReactionMechanismGenerator/T3/context:python" target="_blank">
+    <img src="https://img.shields.io/lgtm/grade/python/g/ReactionMechanismGenerator/T3.svg?logo=lgtm&logoWidth=18" alt="Language grade: Python">
+</a>
+<a href="https://codecov.io/gh/ReactionMechanismGenerator/T3" target="_blank">
+    <img src="https://codecov.io/gh/ReactionMechanismGenerator/T3/branch/master/graph/badge.svg" alt="Coverage">
+</a>
+<a href="http://opensource.org/licenses/MIT" target="_blank">
+    <img src="http://img.shields.io/badge/license-MIT-brightgreen.svg" alt="MIT license">
+</a>
+<a href="https://www.python.org/" target="_blank">
+    <img src="https://img.shields.io/badge/Python-3.7+-blue.svg" alt="Python">
+</a>
+</p>
+
+---
+
+**Documentation**: <a href="https://reactionmechanismgenerator.github.io/T3/" target="_blank">
+https://reactionmechanismgenerator.github.io/T3/</a>
+
```

Comment:
Just to check, I thought this PR was replacing the previous sphinx documentation format. Will this link be updated to point to the new squidfunk format once merged?

Reply:
Yes - Once merged, we'll see whether the auto-deployment thing works as expected (we cannot test it without merging). If it does, it'll auto-push the updated documentation to the T3 `gh-pages` branch. The web page at https://reactionmechanismgenerator.github.io/T3/ takes its contents from that branch. (The auto-deployment should trigger every time a PR is merged (even if it doesn't update the docs), keeping the online docs always up to date.)

Reply:
Awesome sounds good